src/distilabel/llms/base.py

```diff
@@ -72,11 +74,12 @@ class LLM(RuntimeParametersMixin, BaseModel, _Serializable, ABC):
         description="The kwargs to be propagated to either `generate` or `agenerate`"
         " methods within each `LLM`.",
     )
+    structured_output: Optional[Any] = None
```

Comment:
Can we be more specific on the type hinting of this attribute? or we can't because circular imports, etc?

Reply:
I couldn't due to circular imports, and/or validation from pydantic, would love to improve these if we can find a way